instrumentation_sql_test.go

```diff
@@ -232,6 +232,84 @@ func TestOpenSQLDB(t *testing.T) {
 	})
 }
 
```

Comment:
@Angith Does it make sense to move these tests to a table driven test model?

Reply:
Makes sense. I will do it @sanojsubran

Reply:
Done 👍